src/lib/checkMemberEventAccess.js

```diff
@@ -0,0 +1,60 @@
+import debug from 'debug';
+import { dataSources } from '@thatconference/api';
+import orderLib from '../dataSources/cloudFirestore/order';
+import eventLib from '../dataSources/cloudFirestore/event';
+import constants from '../constants';
+
+const dlog = debug('that:api:events:checkMemberEventAccess');
+const memberLib = dataSources.cloudFirestore.member;
+
+export default function checkMemberEventAccess({ user, eventId, firestore }) {
+  dlog('checkMemberEventAccess called');
+
+  const memberId = user.sub;
+  let orderStore;
+  let eventStore;
+  let memberStore;
+  try {
+    orderStore = orderLib(firestore);
+    eventStore = eventLib(firestore);
+    memberStore = memberLib(firestore);
+  } catch (err) {
+    return Promise.reject(err);
+  }
+
+  const memberFunc = memberStore.get(memberId);
```

Comment:
aren't these 3 promises? should we wrap them up in a promise.all

Reply:
Yes they are look 7 lines down. 
I added the functions to variables first then added them to a Promise.all to be run and resolved.